delete_empty_detection.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) 2021 Ichiro ITS
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+import argparse
+import os
+from tqdm import tqdm
+
+if __name__ == "__main__":
+  parser = argparse.ArgumentParser()
+  parser.add_argument('--data_path', help='path for image data',
+                      type=str, default="data")
+  arg = parser.parse_args()
+
+  data_path = arg.data_path
+
+  # loop every directory in <data_path> directory
+  for name in os.listdir(data_path):
+    target_dir = os.path.join(data_path, name)
+    if os.path.isdir(target_dir):
+      print(f"\nRunning on directory {target_dir}...")
+
+      with open(f"{target_dir}.txt", "r") as file_list_text:
+
+        for everything in tqdm(os.listdir(target_dir)):
+          full_path = os.path.join(target_dir, everything)
```

Comment:
I think it's better to use `file` instead of `everything`

Reply:
ok